var/spack/repos/builtin/packages/oce/package.py

```diff
@@ -44,6 +44,8 @@ class Oce(Package):
 
     variant('tbb', default=True,
             description='Build with Intel Threading Building Blocks')
+    variant('X11', default=False,
+            description='Build with X11 enabled')
```

Comment:
Building with `+X11` probably involves a lot of extra dependencies. Can you check what X11 libs this links to with `ldd -r` and add them to this package?